artemis-cli/pom.xml

```diff
@@ -35,6 +35,11 @@
    <dependencies>
       <dependency>
          <groupId>org.apache.activemq</groupId>
+         <artifactId>artemis-boot</artifactId>
```

Comment:
why do you need the boot here? it should be the opposite.. this would be a circular dependency not so useful.

Reply:
I will remove that, it doesn't need to be there

Reply:
removed